uploaddocs.sh

```diff
@@ -47,10 +47,10 @@ do
 
     # TODO: Product page
     echo "Generating metadata (googleapis.dev)"
-    python -m docuploader create-metadata --name $pkg --version $version --language dotnet --github-repository googleapis/google-cloud-dotnet
+    dotnet run --project /tools/Google.Cloud.Tools.DocUploader create-metadata --name $pkg --version $version --language dotnet --github-repository googleapis/google-cloud-dotnet
```

Comment:
I'd suggest adding `--` after the arguments to dotnet, here and below. So:

```sh
dotnet run --project /tools/Google.Cloud.Tools.DocUploader -- create-metadata --name $pkg --version $version --language dotnet --github-repository googleapis/google-cloud-dotnet
```

It makes it clearer which arguments are expected to be parsed by `dotnet`, and which are expected to be parsed by our code.